<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>409</width>
                <height>132</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Add Renter</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>10</y>
                    <width>46</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>Name:</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>40</y>
                    <width>46</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>Surname:</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="name_input">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>10</y>
                    <width>331</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLineEdit" name="surname_input">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>40</y>
                    <width>331</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QDialogButtonBox" name="buttonBox">
            <property name="geometry">
                <rect>
                    <x>240</x>
                    <y>100</y>
                    <width>156</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="standardButtons">
                <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
            </property>
        </widget>
        <widget class="QLineEdit" name="pesel_input">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>70</y>
                    <width>331</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>20</x>
                    <y>70</y>
                    <width>46</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="text">
                <string>PESEL:</string>
            </property>
        </widget>
    </widget>
    <tabstops>
        <tabstop>name_input</tabstop>
        <tabstop>surname_input</tabstop>
        <tabstop>pesel_input</tabstop>
        <tabstop>buttonBox</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
